Source/Core/DolphinWX/Input/InputConfigDiag.cpp

```diff
@@ -71,31 +71,36 @@ void InputConfigDialog::ConfigExtension(wxCommandEvent& event)
   {
   case WiimoteEmu::EXT_NUNCHUK:
   {
-    NunchukInputConfigDialog dlg(this, m_config, _("Nunchuk Configuration"), m_port_num);
+    NunchukInputConfigDialog dlg(this, m_config, _("Nunchuk Configuration"), this->device_cbox,
```

Comment:
`this->` can be omitted, unless it's being used to qualify against two variables with the same name.